lib/Dialect/FIRRTL/FIRRTLIntrinsics.cpp

```diff
@@ -447,16 +461,22 @@ class CirctVerifConverter : public IntrinsicConverter {
   using IntrinsicConverter::IntrinsicConverter;
 
   bool check(GenericIntrinsic gi) override {
-    return gi.hasNInputs(1) || gi.sizedInput<UIntType>(0, 1) ||
+    return gi.hasNInputs(1, 2) || gi.sizedInput<UIntType>(0, 1) ||
```

Comment:
Could you update FIRRTLIntrinsic.md as well?

Reply:
It seems that generic intrinsics aren't mentioned in `FIRRTLIntrinsics.md`, what did you want me to update?

Reply:
Ah ok. I thought FIRRTL LTL intrinsics are documented there as well as other intrinsics. Then it’s fine not to add documents within the PR.